cypress/objects/loginPage.js

```diff
@@ -0,0 +1,31 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid')
+    }
+
+    getLoginFieldUser() {
+        return cy.get('[id="loginForm.userId"]')
+    }
+    
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+
+    getLoginFieldRole() {
+        return  cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRole(role) {
```

Comment:
You called the other function `getSpecificUserFromDropDown()`, so this could be `getSpecificRoleFromDropdown()`, right? ;)